Assets/Scripts/Player/PlayerController.cs

```diff
@@ -59,13 +59,15 @@ public class PlayerController : MonoBehaviour {
 	private float dashResetTimer = 0.5f;
 
 	public bool AllowControls { set => allowControls = value; }
+
+	public Collider2D GroundedCollider { set => groundedCollider = value; }
+
```

Comment:
Never used.

Reply:
Right, that was the first implementation. Will remove